mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -74,6 +74,25 @@ def TilingInterface : OpInterface<"TilingInterface"> {
           return {};
         }]
       >,
+      InterfaceMethod<
+        /*desc=*/[{
+          Method to return iterator domain position computed by the
+          input operand position.
+        }],
+        /*retType=*/"LogicalResult",
+        /*methodName=*/"getIterDomainTilePositionFromOperandPosition",
```

Comment:
Nit: Maybe rename to `getIterationDomainTileFromOperandTile`?

Reply:
DONE rename to getIterationDomainTileFromOperandTile